python/hsfs/core/feature_group_api.py

```diff
@@ -61,10 +82,9 @@ def get(self, feature_store_id, name, version, fg_type):
         :type name: str
         :param version: version of the feature group
         :type version: int
-        :param fg_type: type of the feature group to return
-        :type version: string
+
         :return: feature group metadata object
-        :rtype: FeatureGroup
+        :rtype: FeatureGroup, SpineGroup, ExternalFeatureGroup
```

Comment:
Return type does not match the type hints. Also is it necessary to have return type written explicitly? will the API doc showing the type hints of the return type?